packages/styles/scss/components/date-picker/_flatpickr.scss

```
@@ -338,22 +338,22 @@
     }
 
     &:hover::after {
-      border-top-color: $interactive-01;
-      border-bottom-color: $interactive-01;
+      border-top-color: $background-brand;
+      border-bottom-color: $background-brand;
```

I think it might make more sense to have these map to the `button-primary` token since these styles appear to be related to the increment up/decrement down buttons

```suggestion
      border-top-color: $button-primary;
      border-bottom-color: $button-primary;
```

---

Good call. It was breaking when i used `$button-primary`. I probably need to find where the import for that comes.